CHANGELOG.md

```diff
@@ -8,11 +8,19 @@
 
 - New styles for accordions used in the primary content area `details`, and when used in widgets or filters `details.accordion--controls`.
 - Added `accordion-styles` mixin for creating alternative accordion styles if required. See `_accordions.scss`.
+- Minor design changes
+- Local navigation font-size changed to be smaller than body copy for better visual hierarchy.
```

Comment:
Not sure this is documented in typography page. More content debt for DESIGN-413